pkg/controller/install/install_controller.go

```diff
@@ -161,3 +162,36 @@ func autoInstall(c client.Client, ns string) (err error) {
 	}
 	return err
 }
+
+func matchesClusterRole(roles []string, cr string) bool {
```

Comment:
It's a bit of a style thing, but I would prefer this function be local to `addSCC...`. See below.